pkg/message/evpn.go

```diff
@@ -75,15 +78,19 @@ func (p *producer) evpn(nlri bgp.MPNLRI, op int, ph *bmp.PerPeerHeader, update *
 				prfx.IPLength = *ip
 				gw := e.GetEVPNGWAddr()
 				// IPv4 should have IPLength set to 32
-				if prfx.IPLength == 32 {
-					prfx.IPAddress = net.IP(e.GetEVPNIPAddr()).To4().String()
+				if prfx.IPLength <= 32 {
+					if e.GetEVPNIPAddr() != nil {
```

Comment:
no reason to execute `e.GetEVPNIPAddr()` twice.

if a := e.GetEVPNIPAddr(); a != nil {
   prfx.IPAddress = net.IP(a).To4().String()
}

Could you please comment or send a reference that IPLength can be less than 32, same comment to ipv6